tagstudio/resources/translations/en.json

```diff
@@ -124,9 +171,56 @@
+    "drop_import.duplicates_choice.plural": "The following {count} files have filenames that already exist in the library.",
+    "landing.open_create_library": "Open/Create Library {shortcut}",
+    "video_player.autoplay": "Autoplay",
+    "json_migration.title": "Save Format Migration: \"{path}\"",
+    "json_migration.info.description": "Library save files created with TagStudio versions <b>9.4 and below</b> will need to be migrated to the new <b>v9.5+</b> format.<br><h2>What you need to know:</h2><ul><li>Your existing library save file will <b><i>NOT</i></b> be deleted</li><li>Your personal files will <b><i>NOT</i></b> be deleted, moved, or modified</li><li>The new v9.5+ save format can not be opened in earlier versions of TagStudio</li></ul>",
+    "json_migration.title.old_lib": "<h2>v9.4 Library</h2>",
+    "json_migration.title.new_lib": "<h2>v9.5+ Library</h2>",
+    "json_migration.start_and_preview": "Start and Preview",
+    "json_migration.finish_migration": "Finish Migration",
+    "json_migration.checking_for_parity": "Checking for Parity...",
+    "json_migration.creating_database_tables": "Creating SQL Database Tables...",
+    "json_migration.migrating_files_entries": "Migrating {entries:,d} File Entries...",
+    "json_migration.migration_complete": "Migration Complete!",
+    "json_migration.migration_complete_with_discrepancies": "Migration Complete, Discrepancies Found",
+    "json_migration.discrepancies_found": "Library Discrepancies Found",
+    "json_migration.discrepancies_found.description": "Discrepancies were found between the original and converted library formats. Please review and choose to whether continue with the migration or to cancel.",
+    "json_migration.discrepancies.sql_entry_id_missing": "[Field Comparison]:\nNEW (SQL): SQL Entry ID not found: {id}",
+    "json_migration.discrepancies.fields": "[Field Comparison]:\nOLD (JSON):{json_fields}\nNEW  (SQL):{sql_fields}",
+    "json_migration.discrepancies.subtag": "[Subtag Parity]:\nOLD (JSON):{json_subtags}\nNEW (SQL):{sql_subtags}",
+    "json_migration.discrepancies.alias": "[Alias Parity]:\nOLD (JSON):{json_aliases}\nNEW (SQL):{sql_aliases}",
+    "json_migration.discrepancies.shorthand": "[Shorthand Parity]:\nOLD (JSON):{json_shorthand}\nNEW (SQL):{sql_shorthand}",
+    "json_migration.discrepancies.color": "[Color Parity]:\nOLD (JSON):{json_color}\nNEW (SQL):{sql_color}",
+    "json_migration.heading.match": "Matched",
+    "json_migration.heading.differ": "Discrepancy",
+    "json_migration.heading.entires": "Entries:",
+    "json_migration.heading.tags": "Tags:",
+    "json_migration.heading.shorthands": "Shorthands:",
+    "json_migration.heading.parent_tags": "Parent Tags:",
+    "json_migration.heading.aliases": "Aliases:",
+    "json_migration.heading.colors": "Colors:",
+    "json_migration.heading.file_extension_list": "File Extension List:",
```

Comment:
as per my count 3 out of 10 headings could be taken from somewhere else.  I think it would be better to have them seperate here (I think in general there is value having keys with duplicate values as it makes it simpler down the line when a string needs to change for one of the multiple uses but not the others; it is obviously more work to translate, but I think Weblate recognizes when the base string is the same and suggests other existing translations for that string, so it should just be copy-pasting)

Reply:
Note to self: The ones that are the same as others are `json_migration.heading.tags`, `json_migration.heading.parent_tags` and `json_migration.heading.aliases`